### PR TITLE
chore: fix function names in comment

### PR DESCRIPTION
**Pull request by fuyangpengqi**
**What type of PR is this?**

> Documentation

**What does this PR do? Why is it needed?**

fix function names in comment

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

**Review by saolyn — DISMISSED** [review]
Thank you

File: beacon-chain/core/helpers/validators.go [hunk 1]
```diff
@@ -413,7 +413,7 @@ func IsEligibleForActivationQueueUsingTrie(validator state.ReadOnlyValidator) bo
 	return isEligibileForActivationQueue(validator.ActivationEligibilityEpoch(), validator.EffectiveBalance())
 }
 
-// isEligibleForActivationQueue carries out the logic for IsEligibleForActivationQueue*
+// isEligibileForActivationQueue carries out the logic for IsEligibleForActivationQueue*
```
Comment (terencechain) [inline beacon-chain/core/helpers/validators.go:416]: this is still spelled wrong

File: beacon-chain/blockchain/execution_engine.go [hunk 2]
```diff
@@ -274,7 +274,7 @@ func (s *Service) pruneInvalidBlock(ctx context.Context, root, parentRoot, lvh [
 	}
 }
 
-// getPayloadAttributes returns the payload attributes for the given state and slot.
+// getPayloadAttribute returns the payload attributes for the given state and slot.
```
Comment (saolyn) [inline beacon-chain/blockchain/execution_engine.go:277]: The function above on line 256 the comment is not related to the function.
`// reportInvalidBlock deals with the event that an invalid block was detected by the execution layer` should be modified to reflect the function itself

**Review by saolyn — APPROVED** [review]

**Comment by prestonvanloon** [comment]
Thank you for your contributions. Unfortunately, we are no longer accepting trivial pull requests such as these. Please see the latest [CONTRIBUTING.md](https://github.com/prysmaticlabs/prysm/blob/develop/CONTRIBUTING.md)

**Pull request closed without merging**